futurenet/supervisor/etc/supervisord.conf.d/friendbot.conf

```diff
@@ -0,0 +1,9 @@
+[program:friendbot]
```

Comment:
We won't run a local friendbot for futurenet, only for standalone do we run that.